style.css

```diff
@@ -0,0 +1,332 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins&display=swap');
+
+
+html, body {
+    margin: 0;
+    padding: 0;
+  }
+
+header > div {
+    font-family: "open sans";
+    font-size: 0.5rem;
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+    display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	align-items: stretch;
+    align-content: space-around;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    align-items: center;
+    vertical-align: middle;
+}
+
+nav {
+    width: 600px;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+nav a {
+    text-decoration: none;
+    color: black;
+  
+ 
+}
+
+.blackHeart {
+    width: 20px;
+  
+}
+
+.npmrow {
```

Comment:
Please be consistent in the way of naming a class. Perhaps a camelCase would be apt here.